Cargo.toml

```diff
@@ -27,17 +27,18 @@ failure = "0.1.1"
 filetime = "0.2"
 flate2 = "1.0"
 fs2 = "0.4"
-git2 = "0.7.0"
+git2 = "0.7.2"
 git2-curl = "0.8.1"
+libgit2-sys = "0.7.5" # required only for minimal-versions. brought in by git2 and git2-curl.
+libssh2-sys = "0.2.8" # required only for minimal-versions. brought in by libgit2-sys.
```

Comment:
I think I published new versions of libgit2/git2-rs, so could those new versions be used instead of adding synthetic deps here?